rts/lra-examples/lra-jwt/wildfly/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.jboss.narayana.quickstart.rts.lra</groupId>
+    <artifactId>examples-lra-jwt-wildfly</artifactId>
+    <version>5.10.7.Final-SNAPSHOT</version>
+    <name>LRA jwt wildfly Quickstart Examples</name>
```

Comment:
I'm missing the parent pom definition here.

Reply:
okay